### PR TITLE
Add SignOnly and VerifyOnly

### PR DESCRIPTION
**Pull request by yhung119**
- Updated Crypto from the new core-crypto artifact
- Initial implementaiton of SignOnly and VerifyOnly
- Initial Test to check on result

### CODE REVIEW

File: pkg/crypto.go [hunk 1]
```diff
@@ -58,6 +58,17 @@ type PeacemakrSDK interface {
 	//
 	Decrypt(ciphertext []byte) ([]byte, error)
 
+	//
+	// Sign the message. Returns the signed blob upon success, else returns a non-nil error
```
Comment (notasecret) [inline pkg/crypto.go:62]: awesome, maybe should we mention that the blob has the message in it?

**Review by notasecret — APPROVED** [review]
✌️ 🚀 🥁

**Comment by interstellarPotato** [comment]
Looks very good! I think we need two additional functions:
- SignOnlyInDomain()
- VerifyOnlyInDomain()

**Review by interstellarPotato — APPROVED** [review]
LGTM :) Many thanks

**Pull request merged**